dali/ft/fttransform.ipp

```diff
@@ -192,6 +194,7 @@ public:
     virtual bool setPartition(RemoteFilename & remoteInputName, offset_t _startOffset, offset_t _length, bool compressedInput, const char *decryptKey);
     virtual void setInputCRC(crc32_t value);
     virtual offset_t tell();
+    virtual stat_type getStatistic(StatisticKind kind) override { UNIMPLEMENTED; }
```

Comment:
more: This does not look right.  For instance, if you are spraying a file and converting from utf16 to utf8 this will cause the spray to fail.  Should either be ignored, or call processor->getStatistic